tests/extmod/ubinascii_b2a_base64.py

```diff
@@ -20,3 +20,19 @@
 print(binascii.b2a_base64(b"\x7f\x80\xff"))
 print(binascii.b2a_base64(b"1234ABCDabcd"))
 print(binascii.b2a_base64(b"\x00\x00>"))  # convert into '+'
+
+print(binascii.b2a_base64(b"", newline=True))
+print(binascii.b2a_base64(b"f", newline=True))
+print(binascii.b2a_base64(b"fo", newline=True))
+print(binascii.b2a_base64(b"foo", newline=True))
+print(binascii.b2a_base64(b"foob", newline=True))
+print(binascii.b2a_base64(b"fooba", newline=True))
+print(binascii.b2a_base64(b"foobar", newline=True))
+
+print(binascii.b2a_base64(b"", newline=False))
+print(binascii.b2a_base64(b"f", newline=False))
+print(binascii.b2a_base64(b"fo", newline=False))
+print(binascii.b2a_base64(b"foo", newline=False))
+print(binascii.b2a_base64(b"foob", newline=False))
+print(binascii.b2a_base64(b"fooba", newline=False))
+print(binascii.b2a_base64(b"foobar", newline=False))
```

Comment:
I don't think all these extra test cases are needed.  The only new thing that needs to be tested is the new keyword argument.  So one test with it set to True, and another with it set to False, would be enough IMO.